Gemfile

```diff
@@ -10,6 +10,8 @@ group :development do
   gem 'stringex', '2.8.6'
   # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
   gem 'sassc', '2.1.0'
+  # 1.80.5 breaks the build
+  gem 'sass-embedded', '1.80.4'
```

Comment:
This only breaks when json 2.7.3 is in use.
On current that has been updated to 2.7.4.

Reply:
But `current` has been merged into `next` this morning, so `next` is also running 2.7.4. Or am I missing something?

Reply:
Ah ok, maybe my test was flawed then 👍